docs/configuration.md

```diff
@@ -557,6 +557,8 @@ update-check-delay = 0
 
 [custom]
 ca_bundle = C:\certificates\terena-ssl.crt
+cert_file = C:\certificates\client-ssl.crt
+cert_key_file = C:\certificates\client-ssl.key
```

Comment:
The real documentation is missing, see how other options are explained in that file.